azure/README.md

```diff
@@ -0,0 +1,73 @@
+<!--
+Copyright (c) 2012 - 2016 YCSB contributors. All rights reserved.
```

Comment:
The Travis CI tells that there're errors about SolrClientCloudTest and it can't pass JDK:openjdk7. However, I just made several minor changes.
Could you kindly please check that. Thanks~!